### DISCUSSION TITLE
Why they show the same float in Python?

### DISCUSSION THREAD

**Discussion by 12Jesse24** [1101-GS4719-C/Python-Community-GS4719-C#1636] [Q&A]
a = 23.1415926
print("%3.2f" %a)
print("%2.2f" %a)
print("%1.2f" %a)

When it runs, the first integer behind "%" must be it shows how many integer or "." ,and the second integer must be it shows how many integer behind the "." , but why the example show the same float?

**Comment by BOSENCHANG2004** [comment]
Because the m (field width) is just the minimum width the value will take.
If your number is wider than the minimum, Python will ignore m and print the full number anyway.
